### ISSUE TITLE
Analytics

### ISSUE THREAD

**Issue by yujiri8** [yujiri8/yujiri.xyz#14]
Google search console is a great asset, but search traffic is only part of the picture. I should have my own analytics. (Logs were added recently in preparation for this.)

[Chart.js](https://www.chartjs.org/) might be the way to go for visualizing it. (No way I am touching that Elastic/Kibana pile of crap again.)

**Comment by yujiri8** [comment]
Further work on this should probably be deferred until after #13.

**Comment by yujiri8** [comment]
Don't want this anymore.
